translations/sailfishos-chum-gui.ts

```diff
@@ -378,6 +380,17 @@ Please remove all defined Chum repositories and restart GUI. GUI will add missin
         <source>Use testing repository</source>
         <translation type="unfinished"></translation>
     </message>
+    <message id="chum-settings-advanced">
+        <location filename="../qml/pages/SettingsPage.qml" line="68"/>
+        <source></source>
+        <oldsource>Advanced</oldsource>
+        <translation type="unfinished"></translation>
+    </message>
+    <message id="chum-settings-override-release">
+        <location filename="../qml/pages/SettingsPage.qml" line="78"/>
+        <source>"Override Chum Repository Release"</source>
+        <translation type="unfinished"></translation>
+    </message>
```

Comment:
Those strings have to be specified in QML code via `//% "Text message ..."`. Right now regeneration of TS will drop these strings